content/html/encyclopedia/tables.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
The title should be `tables-in-html.md`. I've changed [the documentation file ](https://github.com/Codecademy/codepedia-content/blob/main/documentation/content-standards.md#standards-for-the-file-name)to reflect this convention, but it was a recent change, so no worries at all for missing it.